service/controller/resource/tcnp/create.go

```diff
@@ -130,12 +132,29 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return microerror.Mask(err)
 		}
 
-		if scale || update {
+		if scale {
 			err = r.updateStack(ctx, cr)
 			if err != nil {
 				return microerror.Mask(err)
 			}
 		}
```

Comment:
I want to keep the scaling logic out of update logic so scaling is possible when some bad stuff happens to tccpn stack

Reply:
Just to make sure, this here is the TCNP stack.

Reply:
Ok below I see now what you meant. In case something bad happened to the TCCPN stack the TCNP stack would still be able to scale.

Reply:
Yeah exactly